packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -75,6 +93,79 @@ export default function Sidebar({
     return <></>;
   }
 
+  if (useNewMobileLayout && !isLaptop && !isMobile) {
```

Comment:
Would this mean we want this for mobile? `!isLaptop && !isMobile` Can we check just isTablet instead?

Reply:
Also I see the useNewMobileLayout already excludes isLaptop, meaning we don't have to check for it is we have useNewMobileLayout

Reply:
> Would this mean we want this for mobile? !isLaptop && !isMobile Can we check just isTablet instead?

We only want this on resolutions above mobile but below tablet, passing just `isTablet` will make it show up on laptop resolutions as well

Reply:
I really don't like that we're using only min values...in that case I think we need to name variables like isTabletAndAbove 😢 else it's confusing.

Anyways in this case u can check for isTablet only since u're already checking for `!isLaptop` in `useNewMobileLayout`

Reply:
> I really don't like that we're using only min values...in that case I think we need to name variables like isTabletAndAbove 😢 else it's confusing.

I agree, at the same time other options can become verbose. I think that as long as we know how these function, it's OK?